src/common/utils.js

```diff
@@ -10,6 +12,34 @@ export const getVerificationStatus = (active, approved) => {
   }
 };
 
+export const setOrganizationFilter = (filter, orgId, orgList) => {
+  // if orgId has a value filter by orgId and sub-orgs, don't include null org ids
+  if (filter.organization_id === ALL_ORGANIZATIONS && orgId) {
```

Comment:
My understanding from the `if` clauses in this function is that only one can be true.
To make that a bit clearer, I'd suggest a chain of `if ... else if ...`

Reply:
It's tricky to come up with variable names.  The filter.organization_id is the filter selected and the "orgId" is the uuid of the logged in user's organization.  

I'll write a comment, but I'm open to better suggestions.

Reply:
How about `userOrgId`?